### PR TITLE
eslint configuration

### PR DESCRIPTION
**Pull request by ahmadysalah**
fix the problem and add eslint and the prettier .

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "editor.formatOnSave": true,
```
Comment (Amoodaa) [inline .vscode/settings.json:2]: format on save: false, bc eslint does the formatting for you using the eslint-config-prettier and plugin

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+  "editor.tabSize": 2,
+  "editor.codeActionsOnSave": {
+    "source.fixAll": true,
+    "prettier.singleQuote": true,
```
Comment (Amoodaa) [inline .vscode/settings.json:6]: as a result this line is obsolete 
https://github.com/Amoodaa/react-context-workshop/blob/master/.vscode/settings.json

File: .eslintrc.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
+module.exports = {
+  extends: ["airbnb", "prettier", "prettier/react"],
```
Comment (Amoodaa) [inline .eslintrc.js:2]: You need to add "airbnb/hooks" in the array

File: .vscode/settings.json (hunk 2 above)
Comment (ahmadysalah) [reply .vscode/settings.json:6]: thanks, a lot Emad , we are stack on this from the morning.

**Pull request merged**